.goreleaser.yaml

```diff
@@ -50,4 +50,4 @@ dist: build
 env:
   - GO111MODULE=on
   - CGO_ENABLED=0
-  - LS_PROTOCOL_VERSION=13
+  - LS_PROTOCOL_VERSION=14
```

Comment:
do we tickets to update Eclipse, IntelliJ & VSCode to that version already?

Reply:
Also, how do we handle protocol version 13 (which hasn't been released in stable either)

Reply:
yeah we will need to skip in both of them

Reply:
we already PRs to bump to 13 for intellij and eclipse. we can adapt these prs